x-pack/plugins/lists/public/exceptions/components/builder/entry_renderer.tsx

```diff
@@ -175,6 +178,22 @@ export const BuilderEntryItem: React.FC<EntryItemProps> = ({
     [onChange, entry]
   );
 
+  const [extendedField, setExtendedField] = useState<DataViewField | null>(null);
+  useEffect(() => {
+    if (!entry.field?.name) {
+      setExtendedField(null);
+    }
+    const fetchExtendedField = async (): Promise<void> => {
+      const fieldName = entry.field?.name;
+      if (getExtendedField && fieldName) {
+        const extendedFields = await getExtendedField([fieldName]);
+        const field = extendedFields.find((f) => f.name === fieldName) ?? null;
+        setExtendedField(field);
+      }
+    };
+    fetchExtendedField();
+  }, [entry.field?.name, getExtendedField]);
```

Comment:
thanks for explaining and adding to notes.
How comparably larger could be response when `include_unmapped=true`?
8k items seems quite large already

Reply:
It can be a really huge difference. Cannot tell exactly in numbers, but what happens when we add `include_unmapped` info is:
- For each field that is unmapped in some indices we would receive `conflictDescriptions` with the whole list of available indices (some will be in `"unmapped"` array and the rest in the `"field_type"` array).

Let's say we have 8k fields and we search through 100 indices. If half of the fields are unmapped in some of the indices for whatever reason (indices store similar but different type of data; we add with time new fields etc.), we will end up with `4k fields * 100 indices` extra lines in the response.